src/components/molecules/ProfilePictureInput/ProfilePictureInput.tsx

```diff
@@ -95,8 +95,14 @@ export const ProfilePictureInput: React.FunctionComponent<ProfilePictureInputPro
     setValue("pictureUrl", pictureUrlRef, true);
   };
 
+  const uploadProfilePic = useCallback((event) => {
+    event.preventDefault();
+    uploadRef.current?.click();
+  }, []);
```

Comment:
Why do you want to stop propagating the click?

Reply:
Because it will submit if we use the keyboard. Am I right? The button trigger the Form submit system.

Reply:
Ah, okay, yeah, makes sense